driver/src/driver/stablex_driver.rs

```diff
@@ -57,23 +57,20 @@ impl<'a> StableXDriver<'a> {
             .auction_orders_fetched(batch_to_solve, &get_auction_data_result);
         let (account_state, orders) = get_auction_data_result?;
 
+        self.past_auctions.insert(batch_to_solve);
```

Comment:
Can we remove the duplicate insert in line 114?

Reply:
Oops! missed that.